doc/zephyr/conf.py

```diff
@@ -1,353 +1,60 @@
-from sphinx.highlighting import lexers
-lexers['DTS'] = DtsLexer()
-
-# A list of ignored prefixes for module index sorting.
-#modindex_common_prefix = []
-
-# If true, keep warnings as "system message" paragraphs in the built documents.
-#keep_warnings = False
-
-# If true, `todo` and `todoList` produce output, else they produce nothing.
-todo_include_todos = False
-
-rst_epilog = """
-.. include:: /substitutions.txt
-"""
-
-# -- Options for HTML output ----------------------------------------------
-
-html_theme = "zephyr"
-html_theme_path = ['{}/doc/themes'.format(NRF_BASE)]
-html_favicon = '{}/doc/static/images/favicon.ico'.format(NRF_BASE)
-
-if tags.has('release'):  # pylint: disable=undefined-variable
-    is_release = True
-    docs_title = 'Docs / %s' %(version)
-else:
-    is_release = False
-    docs_title = 'Docs / Latest'
-
-# The name for this set of Sphinx documents.  If None, it defaults to
-# "<project> v<release> documentation".
-html_title = "Zephyr Project Documentation"
-
-# This value determines the text for the permalink; it defaults to "¶".
-# Set it to None or the empty string to disable permalinks.
-#html_add_permalinks = ""
-
-# A shorter title for the navigation bar.  Default is the same as html_title.
-#html_short_title = None
-
-# The name of an image file (relative to this directory) to place at the top
-# of the sidebar.
-#html_logo = None
-
-# The name of an image file (within the static path) to use as favicon of the
-# docs.  This file should be a Windows icon file (.ico) being 16x16 or 32x32
-# pixels large.
-#html_favicon = None
-
-# Add any paths that contain custom static files (such as style sheets) here,
-# relative to this directory. They are copied after the builtin static files,
-# so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ['{}/doc/static'.format(NRF_BASE),
-                    '{}/doc/static'.format(ZEPHYR_BASE)]
-
-# Add any extra paths that contain custom files (such as robots.txt or
-# .htaccess) here, relative to this directory. These files are copied
-# directly to the root of the documentation.
-#html_extra_path = []
-
-# If not '', a 'Last updated on:' timestamp is inserted at every page bottom,
-# using the given strftime format.
-html_last_updated_fmt = '%b %d, %Y'
-
-# If true, SmartyPants will be used to convert quotes and dashes to
-# typographically correct entities.
-#html_use_smartypants =
-
-# Custom sidebar templates, maps document names to template names.
-#html_sidebars = {}
-
-# Additional templates that should be rendered to pages, maps page names to
-# template names.
-#html_additional_pages = {}
-
-# If false, no module index is generated.
-html_domain_indices = False
-
-# If false, no index is generated.
-html_use_index = True
-
-# If true, the index is split into individual pages for each letter.
-html_split_index = True
-
-# If true, links to the reST sources are added to the pages.
-#html_show_sourcelink =
```

Comment:
We need this set to true to keep the link to the sources.

Reply:
fixed